src/commands/birthday.js

```diff
@@ -27,14 +27,16 @@ module.exports = {
         console.log(member)
         if(member){
             let memberIndex = dbServer.members.findIndex(m => m.user === message.author.tag)
-            dbServer.members[memberIndex].birthday = date       
+            dbServer.members[memberIndex].birthday = date
+            dbServer.members[memberIndex].discord_id = message.author.id     
```

Comment:
discord_id is neccessary for the bot to properly tag the user when wishing them a happy birthday so I updated the birthday.js file to add the users discord id when they create their birthday